src/runtime/particle-execution-context.ts

```diff
@@ -43,6 +43,7 @@ export class ParticleExecutionContext {
   private readonly pendingLoads = <Promise<void>[]>[];
   private readonly scheduler: StorageProxyScheduler = new StorageProxyScheduler();
   private readonly keyedProxies: Dictionary<StorageProxy | Promise<StorageProxy>> = {};
+  private readonly keyedSlots: Map<string, SlotProxy> = new Map();
```

Comment:
It appears like this is keyed by a slot name, as it appears in the particle spec. This is problematic, as PEC can have multiple running particles that use the same slot name in their spec.